lib/email_tracking_retriever.py

```diff
@@ -12,12 +13,15 @@
 import lib.email_auth as email_auth
 from lib import util
 from lib.tracking import Tracking
+from lib.config import open_config
 
 _FuncT = TypeVar('_FuncT', bound=Callable)
 
 BASE_64_FLAG = 'Content-Transfer-Encoding: base64'
 TODAY = datetime.date.today().strftime('%Y-%m-%d')
 MAX_ATTEMPTS = 2
+CONFIG = open_config()
```

Comment:
The config is passed in to the EmailTrackingRetriever object so we can just use `self.config`. 

Of course, ideally we'd just have one single universal config reference -- but for now we should just use self.config

Reply:
I've tried this but was getting some errors, even tried to set it up in the init and pass it in to log_in_if_necessary but continued to get errors, am I missing something?

Reply:
you should be able to just directly reference self.config anywhere in EmailTrackingRetriever